### PR TITLE
Add coding standards for Paradox Syntax files

### PR DESCRIPTION
**Pull request by nntoan**
In this PR, I applied the JSON coding standards for our Paradox script files.

You can check the `.editorconfig` below

```
root = true

[*]
indent_style = space
indent_size = 2
end_of_line = lf
charset = utf-8
trim_trailing_whitespace = true
insert_final_newline = true

[*.txt]
indent_size = 2
```

### Why we need coding standards?

An obvious answer, I'm tired when anytime open a Paradox file which is the combination of 8 and 4 tab-size indents, spaces and tabs...etc

### Requirements

Visual Studio Code is recommended but Atom or Sublime Text is fine too.

- VSCode Extensions:
   * EditorConfig
   * Paradox Syntax
   * Paradox Formatter

### CODE REVIEW

**Comment by nntoan** [comment]
@simsulla Yeah, I do notice a lot of files mess up the encoding after converting to `utf-8`. However, I do notice even in the original files of NWO it's "`?`" already

https://github.com/simsulla/nwomod/pull/1206/files#diff-6334ed0850cb30aa1bf1c649bdb3d630L114

For your other concerns about `*.code-workspace` & `.editorconfig`, fortunately we are using Git 2.0 and as far as my understanding during the whole time playing with GitHub, they are supporting `.gitattributes` very well.

In short, if you want to ignore something from the release zip, add it to `.gitattributes` :)

`.editorconfig` is a **MUST** if we want to keep the coding standards for those files.

**Comment by simsulla** [comment]
Point taken on the *.code-workspace & .editorconfig. On the utf/8 vs ansi encoding, I suggest excluding the **localisation** directory and **common/countries** form the format update. It is up to the developers to never use non ascii characters in the formated files.
